.buildkite/pipeline.trigger.compliance.tests.sh

```diff
@@ -26,7 +26,10 @@ steps:
 EOF
 
 # Generate each test we want to do.
-compliance_test 8.14.0-SNAPSHOT 3.1.5
+## Requires changes in elastic-package to be able to test log_synthetic_mode test package
+# compliance_test 8.15.0-SNAPSHOT 3.2.0
```

Comment:
Could we comment out or skip (not sure if there was something to skip) the feature giving problems? This way we could test the rest of cases with 8.15.

Reply:
I've tried to add support to skip scenarios using a new tag `@skip` in the required scenario.

The steps are shown as skipped but the scenario is shown as PASSED
```
6 scenarios (6 passed)
18 steps (15 passed, 3 skipped)
1m12.182607792s
--- PASS: TestSpecCompliance (72.20s)
    --- PASS: TestSpecCompliance/Integration_package_can_be_installed (15.27s)
    --- PASS: TestSpecCompliance/Input_package_can_be_installed (15.02s)
    --- PASS: TestSpecCompliance/Package_uses_the_"counted_keyword"_type (14.01s)
    --- PASS: TestSpecCompliance/Installer_leverages_runtime_parameter (14.01s)
    --- PASS: TestSpecCompliance/Installer_leverages_subobjects_false (13.87s)
    --- PASS: TestSpecCompliance/Installer_leverages_source_true (0.00s)
PASS
ok  	github.com/elastic/package-spec/compliance	72.914s

```